modules/installation-vsphere-steal-clock-accounting.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+// * installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vmc/installing-restricted-networks-vmc.adoc
+// * installing/installing_vmc/installing-vmc-customizations.adoc
+// * installing/installing_vmc/installing-vmc-network-customizations.adoc
+// * installing/installing_vmc/installing-vmc.adoc
+
+:_content-type: CONCEPT
+[id="installation-vsphere-steal-clock-accounting_{context}"]
+= Steal clock accounting
+
+By default, the installation program provisions the cluster's virtual machines without enabling the steal clock accounting parameter (`stealclock.enabled`). Enabling steal clock accounting can help with troubleshooting cluster issues. After the cluster is deployed, you can use the vSphere Client to enable this parameter on each of the virtual machines.
+
+For more information, see this link:https://https://access.redhat.com/solutions/302283[Red Hat knowledge base article].
```

Comment:
the link seems incorrect, duplicated "https://".

Reply:
Nice catch. I looked right past this.

Reply:
Given that we want our users to do this, I strengthened the language of enabling the parameter. I removed `you can` ans simply stated the vSphere client should be used to enable the parameter.